### PR TITLE
Generate array of ints, not of 1-char strings

### PR DESCRIPTION
**Pull request by silviubogan**
I tested the generator script and also the example, everything works well with the new code.

### CODE REVIEW

File: lib/generate-json.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const fs = require("fs")
 
 fs.writeFileSync(`${__dirname}/pi.json`, JSON.stringify(
-    fs.readFileSync(`${__dirname}/pi.txt`, "utf8").trim().slice(2).split("")
+    fs.readFileSync(`${__dirname}/pi.txt`, "utf8").trim().slice(2).split("").map((x) => parseInt(x))
```
Comment (IonicaBizau) [inline lib/generate-json.js]: `.map(Number)` would do the same, but it shorter :smile:

**Review by IonicaBizau — CHANGES_REQUESTED** [review]
Yup, this will save some space on the disk!
Thank you.

Maybe add a note in the docs mentioning they are numbers.

**Comment by silviubogan** [comment]
1. JSDoc comments to the two exported properties, then regenerate DOCUMENTATION.md.
2. In the JSDoc comment of the `decimals` property specify that it is an array of numbers, not of strings.
3. Recommend using [this](https://github.com/vim-scripts/LargeFile) to open pi.json and pi.txt files in VIM and specify a setting I had to do in my `.vimrc` to make this plugin work for me:

```
let g:LargeFile = 1
```

and also say that the default 100 did not work for me.

@IonicaBizau Do you agree with all of these 3 tasks that I want to work upon?

**Comment by IonicaBizau** [comment]
1. This module is really small (in terms of code), so mentioning in the example, should be enough.
3. Those files shouldn't be edited in any editor.

**Comment by silviubogan** [comment]
I think the changes in 70a2e44 and 9b9e2c3 suffice.

**Review by IonicaBizau — COMMENTED** [review]
👍

**Pull request merged**